test/gtest/conv3d_test.cpp

```diff
@@ -40,7 +40,9 @@ static bool IsTestRunWith(const char* float_arg)
 {
     assert(float_arg != nullptr);
     const auto& s_envVar = miopen::GetStringEnv(ENV(MIOPEN_TEST_FLOAT_ARG));
-    return (!s_envVar.empty() && std::strcmp(s_envVar.c_str(), float_arg) == 0);
+    return ((!s_envVar.empty() && std::strcmp(s_envVar.c_str(), float_arg) == 0 &&
+             miopen::IsEnabled(ENV(MIOPEN_TEST_ALL))) ||
+            (s_envVar.empty() && miopen::IsUnset(ENV(MIOPEN_TEST_ALL))));
```

Comment:
[Recommendation] I think this design looks cleaner:
```suggestion
    if(miopen::IsUnset(ENV(MIOPEN_TEST_ALL)))
        return true; // standalone run
    return miopen::IsEnabled(ENV(MIOPEN_TEST_ALL)) //
        && miopen::GetStringEnv(ENV(MIOPEN_TEST_FLOAT_ARG)) == float_arg;
```
line 42 is not needed anymore.

Reply:
@xinlipn Please see updated https://github.com/ROCm/MIOpen/issues/1843#issue-1420976061 for more info.